backend/src/index.ts

```diff
@@ -0,0 +1,168 @@
+import { ApolloServer } from '@apollo/server'; // preserve-line
+import { startStandaloneServer } from '@apollo/server/standalone'; // preserve-line
+import dotenv from 'dotenv';
+// Prisma
+import { PrismaClient } from '@prisma/client'
+const prisma = new PrismaClient() 
+
+// Password security
+import bcrypt from 'bcryptjs';
+import passport from 'passport';
+import { Strategy as LocalStrategy } from 'passport-local';
+import JwtStrategyConfiguration from '../utils/passportSetup.js';
+import jwt from 'jsonwebtoken';
+
+dotenv.config();
+
+// JWT SETUP
+import { Strategy as JwtStrategy, ExtractJwt } from 'passport-jwt';
+
+const opts: any = {};
+opts.jwtFromRequest = ExtractJwt.fromAuthHeaderAsBearerToken();
+opts.secretOrKey = process.env.SECRET;
+passport.use(JwtStrategyConfiguration);
+
+// A schema is a collection of type definitions (hence "typeDefs")
+// that together define the "shape" of queries that are executed against
+// your data.
+const typeDefs = `#graphql
+  # Comments in GraphQL strings (such as this one) start with the hash (#) symbol.
+
+  # This "Book" type defines the queryable fields for every book in our data source.
+  type Book {
+    title: String
+    author: String
+  }
+
+  type User {
+    email: String!
+    name: String!
+    phone_number: String!
+    address: String!
+    post_code: String!
+    municipality: String!
+  }
+
+  type ValidLogin {
+    token: String!
+    message: String!
+  }
+
+  type Mutation {
+    makeUser (
+    email: String!
+    name: String!
+    phone_number: String!
+    address: String!
+    post_code: String!
+    municipality: String!
+    password: String!
+  ): User
+  
+  }
+
+  type Mutation {
+    loginUser (
+    email: String!
+    password: String!
+  ): ValidLogin
+  
+  }
+
+  # The "Query" type is special: it lists all of the available queries that
+  # clients can execute, along with the return type for each. In this
+  # case, the "books" query returns an array of zero or more Books (defined above).
+  type Query {
+    books: [Book]
+  }
+`;
+
+const books = [
+  {
+    title: 'The Awakening',
+    author: 'Kate Chopin',
+  },
+  {
+    title: 'City of Glass',
+    author: 'Paul Auster',
+  },
+];
+
+
+const resolvers = {
```

Comment:
Move all resolvers to a seperate file from index.ts. This file will otherwise become quite bloated.